### PR TITLE
docs(user): improve WS tutorial

### PR DESCRIPTION
**Pull request by CaselIT**
Some improvements to the ws tutorial

Fixes: #2255
Fixes: #2256

### CODE REVIEW

**Comment by CaselIT** [comment]
@vytas7 I've added an interface only to the simpler echo ws endpoint, since I did not want to overcomplicate the html.

Let me know if you think it's enough

**Comment by CaselIT** [comment]
note to self: we should probably replace or integrate the link at
https://github.com/falconry/falcon/blob/cbca63dc7739720eab856f48b32f9e782438be7a/falcon/asgi/ws.py#L239

with https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent/code

File: docs/user/tutorial-websockets.rst [hunk 1]
```diff
@@ -460,6 +461,7 @@ Things we've changed:
 - Added a new middleware class `AuthMiddleware` that will check the token on the first message.
 - Opening a websocket connection is now handled by the middleware.
 - The client now sends a token as the first message, if required for that route.
+- Falcon was configured to server a simple html page to use the echo websocket client for a browser.
```
Comment (vytas7) [inline docs/user/tutorial-websockets.rst:464]: ~~to server~~ (sic) :arrow_right: _to serve_

Comment (vytas7) [inline docs/user/tutorial-websockets.rst:464]: Maybe capitalize HTML and WebSocket?

File: examples/wslook/wslook/static/index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+<body>
+    <h1>WebSocket Echo Test</h1>
+    <form id="messageForm">
+        <input type="text" id="message" placeholder="Type your message here">
+        <input type="submit" value="Send">
+    </form>
+    <pre id="history"></pre>
+</body>
+
+</html>
```
Comment (vytas7) [inline examples/wslook/wslook/static/index.html:80]: No newline at the end of file

**Review by vytas7 — CHANGES_REQUESTED** [review]
Thanks! I wrote down a couple of comments inline.

Another thing we would need to go through is naming, it seems the tutorial directory is called `wslook`, but in the tutorial itself it is called differently.

**Comment by CaselIT** [comment]
> the tutorial directory is called wslook, but in the tutorial itself it is called differently.

Do you habe a suggestion regarding the new name?

**Comment by vytas7** [comment]
> > the tutorial directory is called wslook, but in the tutorial itself it is called differently.
> 
> Do you habe a suggestion regarding the new name?

I don't have a particular preference, I just find it a bit confusing and error-prone that the same directory is called `wslook` in our repository, and [`asyncws`](https://falcon.readthedocs.io/en/latest/user/tutorial-websockets.html#first-steps) in the tutorial itself.

**Comment by CaselIT** [comment]
makes sense. will think what of the two names makes more sense

**Review by vytas7 — APPROVED** [review]
LGTM, thanks! :100:

**Pull request merged**